backend/eth/client.go

```diff
@@ -243,6 +243,7 @@ func (c *Client) DeployService(
 		return c.deployService(ctx, id, req)
 	})
 	if err != nil {
+		c.logger.Warn(ctx, "DeployService failed in backend/eth/client", err.(errors.Err))
```

Comment:
We shouldn't need to include `backend/eth/client` here -- the logger should itself be passed in the object/package it's logging for on creation and include it with every log (which I believe happens, might want to confirm).

Reply:
oops good catch - meant to take this out